camligorkem/assignment2/PrintAncestorTest.java

```diff
@@ -0,0 +1,64 @@
+import java.util.Arrays;
+import static org.junit.Assert.*;
```

Comment:
Try to avoid importing a whole library: you might end up double importing some classes and having conflicts. It is a much better practice to explicitly import only what you need from a package.

Reply:
done!